### PR TITLE
SDK Changes

### PR DESCRIPTION
**Pull request by Riddhiagrawal001**
## Type of Change

- [ ] Bugfix
- [ ] New feature
- [X] Enhancement
- [ ] Refactoring
- [ ] Dependency updates
- [ ] Documentation
- [ ] CI/CD

## Description
SDK Ui changes 

## Checklist

- [X] I ran `npm run re:build`
- [ ] I reviewed submitted code
- [ ] I added unit tests for my changes where possible
- [ ] I added a [CHANGELOG](/CHANGELOG.md) entry if applicable

### CODE REVIEW

File: src/screens/HyperSwitch/Home/CommonConnectorFlow/ConfigureConnector.res [hunk 1]
```diff
@@ -259,6 +259,7 @@ let make = (~connectProcessorValue: connectProcessor) => {
               amount=100
               returnUrl={`${HSwitchGlobalVars.hyperSwitchFEPrefix}/quick-start`}
               onProceed={updateTestPaymentEnum}
+              keyValue=""
```
[the remark this answers is not in the document]
Comment (Riddhiagrawal001) [reply src/screens/HyperSwitch/Home/CommonConnectorFlow/ConfigureConnector.res]: No we don't need to pass the empty string as we need this value to re-render the SDK . Updated with current time

File: src/screens/HyperSwitch/SDKPayment/TestPayment.res [hunk 2]
```diff
@@ -66,80 +72,92 @@ let make = (
...
+              clientSecret=val
+              publishableKey
+              sdkType=ELEMENT
+              paymentStatus
+              currency
+              setPaymentStatus
+              elementOptions
+              paymentElementOptions
+              returnUrl
+              isConfigureConnector={true}
```
Comment (JeevaRamu0104) [inline src/screens/HyperSwitch/SDKPayment/TestPayment.res:138]: Please check whether `isConfigureConnector` is used?

File: src/screens/HyperSwitch/SDKPayment/SDKPage.res [hunk 3]
```diff
@@ -0,0 +1,197 @@
...
+    [
+      ("currency", "United States-USD"->Js.Json.string),
+      ("amount", "100"->Js.Json.string),
+      ("profile_id", defaultBusinessProfile.profile_id->Js.Json.string),
+    ]->Js.Dict.fromArray
+
+  let (profile, setProfile) = React.useState(_ => defaultBusinessProfile.profile_id)
+  let (amount, setAmount) = React.useState(() => 10000)
+
+  React.useEffect1(() => {
```
Comment (JeevaRamu0104) [inline src/screens/HyperSwitch/SDKPayment/SDKPage.res:99]: Please check this useEffect1

File: src/screens/HyperSwitch/SDKPayment/SDKPaymentTypes.res [hunk 4]
```diff
@@ -0,0 +1,5 @@
+type paymentType = {
+  amount: string,
```
Comment (JeevaRamu0104) [inline src/screens/HyperSwitch/SDKPayment/SDKPaymentTypes.res:2]: change this to int type

**Review by JeevaRamu0104 — APPROVED** [review]

File: src/screens/HyperSwitch/SDKPayment/SDKPage.res (hunk 3 above)
Comment (Riddhiagrawal001) [reply src/screens/HyperSwitch/SDKPayment/SDKPage.res:91]: This use effect is needed to show the payment status after the payment has been done

File: src/screens/HyperSwitch/SDKPayment/SDKPaymentTypes.res (hunk 4 above)
Comment (Riddhiagrawal001) [reply src/screens/HyperSwitch/SDKPayment/SDKPaymentTypes.res:2]: Will address the same in the next PR

**Pull request merged**